kernel/src/process_standard.rs

```diff
@@ -258,6 +277,68 @@ impl<C: Chip> Process for ProcessStandard<'_, C> {
         ret
     }
 
+    /// Transition an unverified process into the State::Unstarted state.
+    /// Should require a capability. -pal
+    fn mark_credentials_pass(
+        &self,
+        credentials: Option<TbfFooterV2Credentials>,
+        short_app_id: Option<process_checking::ShortID>,
+        _capability: &dyn capabilities::ProcessApprovalCapability,
+    ) -> Result<(), ErrorCode> {
+        if self.state.get() != State::Unchecked {
+            Err(ErrorCode::NODEVICE)
+        } else {
+            self.state.update(State::Unstarted);
+            self.app_id.insert(short_app_id);
+            credentials.map(|c| self.credentials.replace(c));
+            Ok(())
+        }
+    }
+
+    fn mark_credentials_fail(&self, _capability: &dyn capabilities::ProcessApprovalCapability) {
+        self.state.update(State::CredentialsFailed);
+    }
+
+    fn get_credentials(&self) -> Option<TbfFooterV2Credentials> {
+        let c = self.credentials.take();
+        self.credentials.insert(c);
+        c
+    }
+
+    /// Enqueue the initialization function of a process onto its task list;
+    /// this is used to start a process. Should only be called when a process
+    /// is in the `State::Unstarted` state.
+    fn enqueue_init_task(
```

Comment:
Please remove this comment and move to below the other enqueue function (match trait).